torch/linalg/__init__.py

```diff
@@ -14,6 +14,52 @@
 # Note: This not only adds doc strings for functions in the linalg namespace, but
 # also connects the torch.linalg Python namespace to the torch._C._linalg builtins.
 
+cross = _add_docstr(_linalg.linalg_cross, r"""
+linalg.cross(input, other, *, dim=-1, out=None) -> Tensor
+
+
+Returns the cross product of vectors in dimension :attr:`dim` of :attr:`input`
+and :attr:`other`.
+
+If :attr:`dim` is not given, it defaults to the last dimension expecting that
+the size of last dimension is 3.
```

Comment:
> Computes the cross product of two 3-dimensional vectors.

I think this can be a little confusing. We can do this with a n-dimensional tensor, just that the cross product is done in the `dim` dimension which has to be 3. Take this `cross` example (holds for `linalg.cross` as well).
```python
a = torch.rand(4,2,1,3)
b = torch.rand(4,2,1,3)
torch.cross(a, b).dim()
>>> 4
```

Reply:
In the writing of the functions in `linalg` (see SVD / QR / ...) we always define them for vectors / matrices etc, and then we mention that we support batches. This way the definitions do not get too cluttered with extra dimensions and slices

Reply:
I see, makes sense to keep it consistent with other `linalg` functions. Updated the doc.